$NetBSD: patch-meson.build,v 1.1 2025/02/14 01:50:54 gutteridge Exp $

Don't build tests that aren't installed, one of which also doesn't compile
under GCC 7.5.

--- meson.build.orig	2025-02-02 19:18:13.000000000 +0000
+++ meson.build
@@ -199,4 +199,3 @@ subdir('libxfce4windowingui')
 subdir('docs' / 'reference' / 'libxfce4windowing')
 subdir('docs' / 'reference' / 'libxfce4windowingui')
 subdir('po')
-subdir('tests')
